NorthstarDLL/filesystem.h

```diff
@@ -24,7 +24,7 @@ OFFSET_STRUCT(VPKFileEntry)
 struct VPKData
 {
 	STRUCT_SIZE(0x50);
-	FIELDS(0x0,
+	STRUCT_FIELD_NOOFFSET(0x0,
```

Comment:
Why? I specifically made the macro so that you could use the same name everywhere. Literally have to remove one line from `structs.h` if this doesnt compile with non-msvc compilers

Reply:
Because using the fields macro with `0x0` causes the preprocessor to try and paste `OFFEST_` and `(` which is invalid under a standards-compliant preprocessor. VS has been warning about this since the standard was changed to 20 (VS' old preprocessor is not standards-compliant) and gcc will refuse to compile this code.